src/components/data_submission/consent_group/EditConsentGroup.js

```diff
@@ -429,45 +429,49 @@ export const EditConsentGroup = (props) => {
           onChange({ key, value: value.dacId });
         }
       }),
+    ]),
+
+    // location
 
-      // location
+    div({style:{ display: 'flex', flexDirection:'row', justifyContent: 'space-between' }}, [
+      h(FormFieldTitle, {
+        required: true,
+        title: 'Data Location',
+        description: 'Please provide the location of your data resource for this consent group',
+      }),
+    ]),
+    div({className: 'flex flex-row'}, [
       h(FormField, {
-        id: idx + '_dataLocation',
+        style: { width: '50%' },
+        id: idx+'_dataLocation',
         name: 'dataLocation',
         type: FormFieldTypes.SELECT,
         isMulti: true,
-        title: 'Data Location',
-        description: 'Please provide the location of your data resource for this consent group',
         exclusiveValues: ['Not Determined'],
         selectOptions: [
-          'AnVIL Workspace',
-          'Terra Workspace',
-          'TDR Location',
+          'Existing AnVIL Workspace',
+          'Existing Terra Workspace',
+          'Existing TDR Snapshot',
           'Not Determined',
+          'I am requesting an AnVIL workspace for storage',
         ],
-        defaultValue: consentGroup.dataLocation,
         placeholder: 'Data Location(s)',
-        validators: [FormValidators.REQUIRED],
+        defaultValue: consentGroup.dataLocation,
         onChange,
         validation: validation.dataLocation,
         onValidationChange,
       }),
-
       h(FormField, {
-        id: idx + '_url',
+        style: { width: '50%', paddingLeft: '1.5%' },
+        id: idx+'_url',
         name: 'url',
-        title: 'Data URL',
-        validators: [FormValidators.REQUIRED, FormValidators.URL],
-        placeholder: 'Free text field for entering URL of data',
+        placeholder: 'Enter a URL for your data location here',
         defaultValue: consentGroup.url,
         onChange,
-        validation: validation.url,
-        onValidationChange,
       }),
     ]),
```

Comment:
One question I do have is if this is ever a required field depending on a selected value in `Data Location`. It's fine for now - we can follow up in the schema definition.

Reply:
I'm going to take out the validation, it shouldn't be required. Thanks for pointing it out!